pravega/bookkeeper-operator
# Issue 70: I/O fails when bookkeeper operator upgrades and sts get's restarted due to change in the ledger mount path in default case

Signed-off-by: prabhaker24 <prabhaker.saxena@dell.com>

### Change log description
I/O fails when bookkeeper operator upgrades and sts get's restarted due to change in the ledger mount path in the default case

### Purpose of the change
Fixes #70 

### What the code does
This code ensures that the ledger MountPath doesn't change in case of default case where we don't specify anyMountPath and upgrade the bookeeper-operator or bookkeeper-cluster.

### How to verify it
1) Install Bookkeeper-operator 0.1.1 and deploy bk cluster along with the pravega cluster and upgrade the operator with this fix and then delete the sts and check if the controller is still able to create stream and scopes.
2) Install bookkeeper cluster 0.5.1 along with pravega cluster with bookkeeper operator upgrade the operator with this fix and check if the controller is still able to create streams and scopes.
